jme3-core/src/main/java/com/jme3/input/dummy/DummyKeyInput.java

```diff
@@ -48,4 +48,9 @@ public int getKeyCount() {
         return 0;
     }
 
+    
+    public String getKeyName(int key){
+        return null;
```

Comment:
might be better to throw an UnsupportedOperationException

Reply:
Not sure. In this case it might be better to return a constant string eg "Unknown" to not break user code with npe or other exceptions in case the code is designed to work with both real and dummy input

Reply:
"Unknown" sounds good.